src/psql/nodo/liquibase/changelog/cfg/db.changelog-master-3.31.0.xml

```diff
@@ -7,6 +7,9 @@
 
   <include file="./db.changelog-master-3.30.0.xml"/>
 
+  <include file="./${version}/db.changelog-20240711000000_add_field_aca.xml" labels="${version}"/>
```

Comment:
@alessio-cialini this is not need for `cfg` its already included. into `db.changelog-master-3.30.0.xml`